pbf/views.py

```diff
@@ -243,7 +243,8 @@ def add_player(request, game_id):
     else:
         starting_energy = spirit_starting_energy[spirit.name]
 
-    gp = GamePlayer(game=game, spirit=spirit, color=colors[0], aspect=aspect, starting_energy=starting_energy)
+    # energy is intentionally set to starting_energy when adding a new player
+    gp = GamePlayer(game=game, spirit=spirit, color=colors[0], aspect=aspect, energy=starting_energy, starting_energy=starting_energy)
```

Comment:
Setting `energy=starting_energy `is incorrect behaviour.

This will cause each spirit to immediately start with its base energy gain PER TURN. For example, River, which has `starting_energy` 1 (denoting that its base energy gain per turn is 1 if it hasn't unconvered any track spaces), will start the game with 1 energy, which is incorrect.